src/main/java/org/zalando/fahrschein/domain/DataOperation.java

```diff
@@ -0,0 +1,28 @@
+package org.zalando.fahrschein.domain;
+
+/**
+ * The type of operation executed on the entity.
+ * <ul>
+ * <li><strong>C</strong>: Creation</li>
+ * <li><strong>U</strong>: Update</li>
+ * <li><strong>D</strong>: Deletion</li>
+ * <li><strong>S</strong>: Snapshot</li>
+ * </ul>
+ */
+enum DataOperation {
```

Comment:
> @JsonProperty("data_op")

Could this be added to the interface instead?

I was thinking about (de-)serializing the enum itself, as `C` instead of `CREATE` for example.